src/prisma/generator/templates/http.py.jinja

```diff
@@ -2,7 +2,11 @@
 # -- template http.py.jinja --
 {% if generator.config.http == 'aiohttp' %}
 from ._aiohttp_http import *
-{% else %}
+{% elif generator.config.http == 'requests' %}
 from ._requests_http import *
+{% elif generator.config.http == 'httpx-async' %}
+from ._httpx_async_http import *
+{# else: generator.config.http == 'httpx-sync' #}
+{% else %}
+from ._httpx_sync_http import *
```

Comment:
Even though it's uninformative, raise an error if the target http library isn't handled here.
```suggestion
{% elif generator.config.http == 'httpx-sync' %}
from ._httpx_sync_http import *
{% else %}
{{ 0 / 0 }}
```